packages/client/modules/userDashboard/components/OrgBilling/OrgPlan.tsx

```diff
@@ -121,6 +121,7 @@ const CTAButton = styled(BaseButton)<{
 }>(({buttonStyle}) => ({
   width: '80%',
   boxShadow: buttonStyle === 'primary' ? Elevation.Z8 : Elevation.Z0,
+  opacity: 1,
```

Comment:
Overwriting the disabled styles in base button. Would be better to use plain buttons here and style them from scratch, but I just wanted to be done with this fix.